src/util/customTransformerFactory.js

```diff
@@ -0,0 +1,48 @@
+const {
+  setLambdaUserTransform,
+  runLambdaUserTransform
+} = require("./customTransformer-lambda");
+
+const {
+  setOpenFaasUserTransform,
+  runOpenFaasUserTransform
+} = require("./customTransformer-faas");
+
+const {
+  userTransformHandlerV1,
+  setUserTransformHandlerV1
+} = require("./customTransformer-v1");
+
+const UserTransformHandlerFactory = userTransformation => {
```

Comment:
What if we move the metrics here?

Avoid repeating them for every integration and enforce consistency that will enable comparison of one provider vs the other.

Reply:
Ignoring here. 
common stats were already captured + some specific stats are added for lambda & faas